README.md

```diff
@@ -933,7 +941,9 @@ There are not many options on the command line, most of the options are in the c
 * **[-h]**: Display quick help
 * **[-c | --config] configuration_file**: Specify a configuration file other than the default
 * **[-f | --format] configuration_format**: Specify the configuration file format: `toml`, `yaml`, `json` or `hcl`
-* **[-n | --name] profile_name**: Profile section to use from the configuration file
+* **[-n | --name] profile_name**: Profile section to use from the configuration file.
+  You can also use `[profile_name].[command]` syntax instead, this will only work if `-n` is not set.
+  Bot using `-n [profile_name] [command]` and `[profile_name].[command]` are supported.
```

Comment:
Small typo in this line. how about:

Using `-n [profile_name] [command]` or `[profile_name].[command]` both select profile and command and are technically equivalent.

Reply:
Thanks!